OracleDatabase/SingleInstance/dockerfiles/19.3.0/createDB.sh

```diff
@@ -70,9 +70,12 @@ DEDICATED_THROUGH_BROKER_LISTENER=ON
 DIAG_ADR_ENABLED = off
 " > $ORACLE_HOME/network/admin/listener.ora
 
+# Directory for storing archive logs
+export ARCHIVELOG_DIR=$ORACLE_BASE/oradata/$ORACLE_SID/$ARCHIVELOG_DIR_NAME
+
 # Start LISTENER and run DBCA
 lsnrctl start &&
-dbca -silent -createDatabase -enableArchive $ENABLE_ARCHIVELOG -archiveLogDest $ORACLE_BASE/oradata/$ORACLE_SID/archive_logs -responseFile $ORACLE_BASE/dbca.rsp ||
+dbca -silent -createDatabase -enableArchive $ENABLE_ARCHIVELOG -archiveLogDest $ARCHIVELOG_DIR_NAME -responseFile $ORACLE_BASE/dbca.rsp ||
```

Comment:
In this line shouldn't it  be $ARCHIVELOG_DIR instead of $ARCHIVELOG_DIR_NAME

Reply:
Sorry, my bad. Resolved in the commit https://github.com/oracle/docker-images/pull/1943/commits/e934a0472bed35111dfc801ad7cc82c864ffd487.